Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/TranslationHandles.cs

```diff
@@ -0,0 +1,338 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControlTypes;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControl
+{
+    /// <summary>
+    /// Translation handles for <see cref="BoundsControl"/> that are used for rotating the
+    /// Gameobject BoundsControl is attached to with near or far interaction
+    /// </summary>
+    public class TranslationHandles : HandlesBase
```

Comment:
Reminder; this is mostly identical to the existing RotationHandles and ScaleHandles. Suggestion for this file should also apply to the other files, too.